api/room.go

```diff
@@ -172,6 +165,24 @@ func (r *Room) join(l *LoginReq, c *websocket.Conn) bool {
 	return !alreadyExists
 }
 
+// Sends message to all users in the room.
+func (r *Room) sendMsg(user string, msg string, saveToHistory bool) {
```

Comment:
remove the saveToHistory parameter since you always pass `true` there

NOTE: `bool` params should be rarely used in practice, since they are hard to reason about at the call site (e.g. you don't know what exactly does "true" or "false" mean unless you look at the function definition). 
A slightly better options are:
1) Make two functions with distinct names, if the code paths for "true" and "false" values of this bool param  are sufficiently different
2) add a comment at each call site: `foo(param1, param2, /*saveToHistory = */ true)` <-- this helps readability, but is ugly from the coder's perspective, since you need to write it at all times
3) If you really want to keep one function and it will be called a lot (so, (1) and (2) do not work) introduce a named enum and use it instead:

```
// this is the syntax for enums in Go
type ChatHistoryPolicy bool
const (
  SaveToHistory ChatHistoryPolicy = true
  SkipHistory ChatHistoryPolicy = false
)

func (r *Room) sendMsg(..., policy ChatHistoryPolicy) ..

// called as
sendMsg(..., SaveToHistory)
sendMsg(..., SkipHistory)
```